crates/aptos/src/move_tool/mod.rs

```diff
@@ -309,7 +314,7 @@ fn compile_move(build_config: BuildConfig, package_dir: &Path) -> CliTypedResult
     // TODO: Add caching
     build_config
         .compile_package(package_dir, &mut Vec::new())
-        .map_err(|err| CliError::MoveCompilationError(err.to_string()))
+        .map_err(|err| CliError::MoveCompilationError(format!("{:#}", err)))
```

Comment:
?

Reply:
Pretty print, interesting...

Reply:
Thats for getting the chain of causes from anyhow. It's a bit of magic but that's how it is described in the anyhow docs.